### PR TITLE
UPDATE: README page description

### PR DESCRIPTION
**Pull request by sksingh2022**
# Description

Documentation ReadMe update

Fixes # No issues but just fixes to the contents

## Type of change

- [x] Documentation (update or new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from ... on the Pull Request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,19 @@ Add all of your Research, Meeting Agendas and everything project related.
 ## Contributing
 
 Please see the [contribution guidelines](CONTRIBUTING.md).
+
+
+## What is this repositry used for?
+
+This documentation repo is used to collate all information in regards to the Thoth-Tech Project.
```
Comment (talzid) [inline README.md:15]:
```suggestion
This documentation repo is used to collate all information regarding Thoth Tech Products 
```

File: README.md [hunk 2]
```diff
@@ -8,3 +8,19 @@ Add all of your Research, Meeting Agendas and everything project related.
...
+This documentation repo is used to collate all information in regards to the Thoth-Tech Project. 
+
+Each team has their own separate repository and these are supposed to be used for anything related to the team projects. Additionally, if you have documentation elsewhere, we recommend that you begin a small migration process to this repository within a .md format.
+
+Some of the things you might want to include within this repositry includes: 
+
+* Meeting Minutes
+* Research and Findings
+* Onboarding Packages
+* And more - the potential is unlimited.
```
Comment (talzid) [inline README.md:24]: Using * as bullet points might be a cause of failing the formatting check

Consider 
- a dash for a list

**Review by talzid — COMMENTED** [review]
Content looks good, just a few minor changes.

A thought to consider would be the longevity of the ReadMe.md. As this file is the opening message of this repository it should be applicable to all audiences with little need to change overtime. 

Otherwise, should be good to merge once formatting is updated. 
Look into the npm formatter guides for installation and use. If you need some assistance give us a shout.

File: README.md (hunk 1 above)
Comment (sksingh2022) [reply README.md]: Hi Talia, No worries at all. Thank you for your comments

**Review by talzid — APPROVED** [review]

**Pull request merged**